configuration/scripts/cice.batch.csh

```diff
@@ -185,7 +185,7 @@ cat >> ${jobfile} << EOFB
 #SBATCH --qos=standby
 EOFB
 
-else if (${ICE_MACHINE} =~ daley* || ${ICE_MACHINE} =~ banting*) then
```

Comment:
Hi Till! I don't really mind that you add your machine here to the same `else if` statement as ours (the PBS directives should not change often), but if you (or I) want to change them in the future, maybe it makes more sense for you to have a separate entry for your machine ?